pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -45,6 +46,8 @@
 #define log_debug(fmt, ...)
 #endif
 
+#define LOAD_CONSTANT(param, var) asm("%0 = " param " ll" : "=r"(var))
```

Comment:
This a bit cryptic. Worth adding some comments!

Reply:
This is the post-compilation constant edition feature of the new lib 🎉 

Essentially what this part does is insert a load instruction with an immediate value, into the variable provided in the output section of the `asm()` call. The inserted symbol is used so that the rewrite function is able to detect where the constant should be edited.
Have a look at https://github.com/DataDog/ebpf/blob/master/manager/editor.go#L26-L65.

Reply:
@Gui774ume very ingenious :)